Foundations JavaScript basics: Format headings with sentence case (#25852)
TheOdinProject/curriculum: foundations/javascript_basics/problem_solving.md

In short, break the big problem down and solve each of the smaller problems until you've solved the big problem.

> ### Solving fizz buzz

Reviewer: ```suggestion
### Solving Fizz Buzz
```

Decided to give it title case because it's a game

> To demonstrate this workflow in action, let's solve a common programming exercise: [FizzBuzz](https://en.wikipedia.org/wiki/Fizz_buzz).

Reviewer: ```suggestion
To demonstrate this workflow in action, let's solve [a common programming exercise: Fizz Buzz, explained in this wiki article](https://en.wikipedia.org/wiki/Fizz_buzz).
```

Updated for consistency with the header and to create a more descriptive link which is unrelated to this PR

> #### Understanding The problem

Reviewer: ```suggestion
#### Understanding the problem
```
